loefsys/events/models/event.py

```diff
@@ -30,7 +35,21 @@ class Event(models.Model):
 
     # description = HTMLField(_("description"),)
 
-    # organiser (should use contenttypes for user, committee, yearclub)
+    organisers = models.ManyToManyField(
+        to=settings.AUTH_USER_MODEL,
+        related_name="event_organisers",
+        blank=True,
+    )
+    
+    contacts = models.ManyToManyField(
```

Comment:
This should be `organiser_contacts`, in the object create function we will check whether either `organiser_groups` or `organiser_contacts` contains at least one entry.